agent/main.go

```diff
@@ -141,10 +141,10 @@ func newCmd() *cobra.Command {
 				}),
 			)
 			gitOpsEngine := engine.NewEngine(config, clusterCache)
-			errors.CheckError(err)
+			errors.CheckErrorWithCode(err, errors.ErrorCommandSpecific)
```

Comment:
Was looking at the code and noticed that this check is not needed. Trying to use the engine...